app/models/notification/right_side_popup.rb

```diff
@@ -0,0 +1,9 @@
+class Notification::RightSidePopup < Notification
```

Comment:
I agree with @allenwq.

Problem: can a notification only be one kind? Can't the notification appear in multiple forms?